### PR TITLE
Feature/design dashboard

### PR DESCRIPTION
**Pull request by audreyhuguenin**

### CODE REVIEW

File: packages/frontend/src/assets/logout.svg [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+</svg>
```
Comment (verdonarthur) [inline packages/frontend/src/assets/logout.svg:53]: Can it be optmized/minimized ? (It's a question, not a demand)

File: packages/frontend/src/components/atoms/Burger.scss [hunk 2]
```diff
@@ -0,0 +1,95 @@
+button.navbar-toggler {
```
Comment (verdonarthur) [inline packages/frontend/src/components/atoms/Burger.scss:1]: Atoms are not scss file but true component, your file here must go in the AdminLayout.scss or if you want to separate it from it, you can also create a folter in /.../template/AdminLayout/ and put all file related to the AdminLayout in it

File: packages/frontend/src/components/molecules/DashboardCard.tsx [hunk 3]
```diff
@@ -1,19 +1,22 @@
 import React from 'react';
+import './DashboardCard.scss';
 
 interface IProps {
-  title: string;
+  value: number;
```
Comment (verdonarthur) [inline packages/frontend/src/components/molecules/DashboardCard.tsx:5]: I don't think we're always going to have a number there ! We could have for example a number with a percentage so i think that string is more adapted here.

**Review by verdonarthur — CHANGES_REQUESTED** [review]

File: packages/frontend/src/assets/logout.svg (hunk 1 above)
Comment (audreyhuguenin) [reply packages/frontend/src/assets/logout.svg:53]: Yep will do that

**Comment by audreyhuguenin** [comment]
Done! :) Other changes?

**Comment by verdonarthur** [comment]
> 
> 
> Done! :) Other changes?

Nope ! For my part alles Gut ! You can merge it 👍

**Pull request merged**